app/controllers/users/confirmations_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Users::ConfirmationsController < Devise::ConfirmationsController
```

Comment:
Why do we need a class, which we are not overriding?

Reply:
Fixed